examples/flava/native/train.py

```diff
@@ -167,34 +193,16 @@ def create_model(self) -> torch.nn.Module:
                 mixed_precision=mp,
                 auto_wrap_policy=partial(
                     transformer_auto_wrap_policy,
-                    transformer_layer_cls={TransformerEncoderLayer},
```

Comment:
can we add a toggle for this?

Reply:
Sure, what are the two choices? It needs some wrapping policy AFAIK.